doc/en/components/grids/grid/overview.md

```diff
@@ -20,7 +20,7 @@ The {Platform} `{GridName}` is used as a feature-rich control for displaying dat
 
 ## {Platform} Grid Example
 
-In this grid example, you can see how users can do both basic and excel-style filtering, live-data sorting, as well as using of grid summaries and cell templating. The demo also includes custom paging and per page usage part of the [Pagination](paging.md).
+In this grid example, you can see how users can do both basic and excel-style filtering, live-data sorting, as well as using of grid summaries and cell templating. The demo also includes custom paging and per page usage part of the Pagination.
```

Comment:
not really "custom paging", just a paginator included, so perhaps just leave out the custom bit. Also not exactly certain on the "per page usage part of the Pagination" - it's copied from the Angular topic but with the platform word removed and it sounds off to me at least. Maybe can be reworded to explain there's a paginator component added for paging or just skip the entire explanation.